SETUP.md

```diff
@@ -32,8 +33,8 @@ Currently, this repository supports **Python CPU**, **Python GPU** and **PySpark
 
 ### Requirements
 
-* A machine running Linux, MacOS or Windows
-* Anaconda with Python version >= 3.6
+* A machine running Linux, MacOS or Windows.
```

Comment:
not a fan of using period at the end of incomplete sentences, it's just an extra character
that said, it really doesn't matter much to me =)